src/components/FormEditClinic.js

```diff
@@ -0,0 +1,336 @@
+    <div className="Addfield" key={id}>
+      <article className="Formfield">
+        <div>
+          <input
+            type="text"
+            name="ClinicName"
+            id="ClinicName"
+            value={clinicName}
+            onChange={(e) => {
+              setClinicName(e.target.value);
+            }}
+            className="InputClinicName"
+          />
+        </div>
+
+        <div className="PlaceInput">
+          <input
+            type="text"
+            name="Place"
+            id="Place"
+            value={place}
+            onChange={(e) => {
+              setPlace(e.target.value);
+            }}
+            className="InputPlace"
+          />
+        </div>
+
+        <div className="InsuranceSelect">
+          <p className="FormTitle"> Versicherungsart</p>
+          <label for="Insurance"> </label>
+          <div className="SelectMenu">
+            <select
+              value={insurance}
+              onChange={(e) => {
+                setInsurance(e.target.value);
+              }}
+            >
+              <option default> Bitte wählen</option>
+              <option value="privat"> Privatversicherung</option>
+              <option value="public"> Gesetzliche Versicherung</option>
+              <option value="both"> beide</option>
+            </select>
+          </div>
+        </div>
+
+        <div className="Therapy">
+          <p className="FormTitle"> Therapie</p>
+          <div className="TherapyCheckbox">
+            <div className="LeftCol">
+              <div className="Checkbox">
+                <input
+                  type="checkbox"
+                  checked={therapy.Kunst}
+                  onChange={(e) =>
+                    setTherapy({ ...therapy, Kunst: e.target.checked })
+                  }
+                />
+                <label for="Kunst"> Kunst </label>
+              </div>
+
+              <div className="Checkbox">
+                <input
+                  type="checkbox"
+                  checked={therapy.Sport}
+                  onChange={(e) =>
+                    setTherapy({ ...therapy, Sport: e.target.checked })
+                  }
+                />
+                <label for="Sport"> Sport </label>
+              </div>
+
+              <div className="Checkbox">
+                <input
+                  type="checkbox"
+                  checked={therapy.Gruppen}
+                  onChange={(e) =>
+                    setTherapy({ ...therapy, Gruppen: e.target.checked })
+                  }
+                />
+                <label for="Gruppen"> Gruppen </label>
+              </div>
+            </div>
+            <div className="CenterCol">
+              <div className="Checkbox">
+                <input
+                  type="checkbox"
+                  checked={therapy.Bewegung}
+                  onChange={(e) =>
+                    setTherapy({ ...therapy, Bewegung: e.target.checked })
+                  }
```

Comment:
please make this sub components for the love of god

Reply:
😂